changes/change_caseToTest.md

```diff
@@ -0,0 +1 @@
+TAT trend report metrics column and row name from Case Count to Test Count, Full Case to Full Test
```

Comment:
```suggestion
TAT Trend Report column and row name from Case Count to Test Count, Full Case to Full Test
```
- proper noun
- part of this may be considered metrics in some context, but none of it is what we refer to as metrics within Dimsum